docs/source/_templates/index.html

```diff
@@ -1,92 +0,0 @@
-{% extends "layout.html" %}
```

Comment:
This file contains content, and is relatively painful to maintain. 
Content should not be in `_template`, `_template`  should just  contain rendering logic.